minifi/minifi-commons/minifi-utils/src/test/java/org/apache/nifi/minifi/commons/status/util/PropertyUtilTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.minifi.commons.status.util;
+
+import static org.apache.nifi.minifi.commons.status.util.PropertyUtil.resolvePropertyValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class PropertyUtilTest {
+
+    private static final String PROPERTY_KEY_WITH_DOTS = "property.foo.bar";
+    private static final String PROPERTY_KEY_WITH_HYPHENS = "property-foo-bar";
+    private static final String PROPERTY_KEY_WITH_DOTS_AND_HYPHENS = "property.foo-bar";
+    private static final String PROPERTY_KEY_WITH_UNDERSCORE = "property_foo_bar";
+    private static final String VALUE = "value";
+
+    @Test
+    void testResolveParameterValueReturnsNullWhenPropertiesDoesNotContainTheKey() {
+        assertNull(resolvePropertyValue(PROPERTY_KEY_WITH_DOTS, Map.of()));
+    }
+
+    @Test
```

Comment:
You can control the test name for the parameters I think as well also with descriptive parameter names it would be self explanatory + for the parameetrized tests

Reply:
Done